app/com/gu/floodgate/AppComponents.scala

```diff
@@ -63,11 +64,23 @@ class AppComponents(context: Context) extends BuiltInComponentsFromContext(conte
   )
 
   val reindexService = new ReindexService(contentSourceService, runningJobService, jobHistoryService, progressTrackerControllerActor, wsClient)
-
-  val contentSourceController = new ContentSourceApi(contentSourceService, reindexService, jobHistoryService, runningJobService)
   val runningJobController = new RunningJobApi(runningJobService)
   val jobHistoryController = new JobHistoryApi(jobHistoryService)
 
+  val previewCodeBulkJobActor = generateBulkActors("draft-code")
```

Comment:
As discussed in person, I chose to have an actor per `stack-stage` so that we could manage state at that level, and permit manual reindexes to be triggered in different stages in parallel to a bulk job.